### PR TITLE
[1120] ui/plus: Integrate cls endpoints

### LINKED ISSUE #1120: Integrate finalized API endpoints
The API has been mocked in cypress, trying to match the in-progress work of https://github.com/ethyca/fidesctl-plus/pull/106. Once that is merged, there are some points of integration we need to check:
- Currently we get all all classify instances in one large list. The list endpoint will only return the instance IDs and statuses. The details will be fetched per dataset fides_key.
-  To integrate the component from #1077, we need to map the classification labels into category objects.
- Any PUT/POST body changes.
- Generate API types from OpenAPI and replace the types defined in `plus.slice.ts`
- Toast messages for field edits: [design](https://www.figma.com/file/MTbZohIGvB4bq2f2wkiDHO/Fidesops-Management-UI?node-id=3527%3A55560)
- This could also be a good place to add documentation.

### PR DESCRIPTION
**Pull request by ssangervasi**
Closes #1120 

### Code Changes

- [x] APIs restructured to match https://github.com/ethyca/fidesctl-plus/pull/106 (in progress)
	- `GET classify/`
	- `POST classify/`
	- `PUT classify/`
- [x] New API introduced for the deep classification details:
	- `GET classify/{dataset_fides_key}`
- [x] Generate API types from OpenAPI and replace the types defined in `plus.slice.ts`
- [x] Map the classification labels into category objects.

### Steps to Confirm

* [x] Testing party with actual databases to classify!
* [ ] Dataset editing with non-plus build should be unchanged (except for "status" column).

### Pre-Merge Checklist

* [x] All CI Pipelines Succeeded
* Documentation Updated:
  - [ ] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)
  - [ ] documentation issue created (tag docs-team to complete issue separately)
* [x] Issue Requirements are Met
* [x] Relevant Follow-Up Issues Created
    - https://github.com/ethyca/fides/issues/1232
* [x] Update `CHANGELOG.md`

### CODE REVIEW

File: clients/ctl/admin-ui/cypress/.eslintrc.json [hunk 1]
```diff
@@ -1,7 +1,11 @@
 {
   "extends": ["plugin:cypress/recommended"],
+  "plugins": ["no-only-tests"],
```
Comment (ssangervasi) [inline clients/ctl/admin-ui/cypress/.eslintrc.json:3]: Being able to use `it.only` is helpful while developing with cypress mocked APIs, but you def don't want to commit them.

**Review by SteveDMurphy — APPROVED** [review]
Understanding my approval doesn't mean as much as others most likely but loved seeing this in action today 🙌🏽

**Review by LKCSmith — APPROVED** [review]
Whew! I think this looks good and tests as expected.

**Comment by ThomasLaPiana** [comment]
merging myself for expediency, thanks for the reviews all!

**Pull request merged**